packages/shared/src/lib/image.ts

```diff
@@ -37,5 +37,17 @@ export const cloudinary = {
   squads: {
     imageFallback:
       'https://daily-now-res.cloudinary.com/image/upload/v1672041320/squads/squad_placeholder.jpg',
+    tour: {
```

Comment:
In preview these all seem low quality?

Reply:
I think that is only on the preview. They look pretty alright on normal browsing.